sdk/iothub/Azure.ResourceManager.IotHub/src/autorest.md

```diff
@@ -9,7 +9,7 @@ csharp: true
 library-name: IotHub
 namespace: Azure.ResourceManager.IotHub
 require: https://github.com/Azure/azure-rest-api-specs/blob/0f9df940977c680c39938c8b8bd5baf893737ed0/specification/iothub/resource-manager/readme.md
-tag: package-2021-07-02
+tag: package-2022-04-30-preview
```

Comment:
Please remove this, if you want to release a track 2, please establish another pr

Reply:
reverted